File: src/main/scala/com/easyday/dao/UvAggrDao.scala
package com.easyday.dao

import com.easyday.domain.LogRecord

/**
 * Created by admin on 2018/4/11.
 * 4/23.迁移值java目录下
 */
trait UvAggrDao {
    def insert(logRecord :LogRecord):Int
    def delete(dt:Int,logtime:Long) :Int
    def truncate (): Unit

}

File: src/main/java/cn/eastday/dao/UvAggrDao.java
package cn.eastday.dao;

import com.easyday.domain.LogRecord;

import java.util.List;

/**
 * Created by admin on 2018/4/11.
 */
public interface UvAggrDao {
    int  insert(LogRecord logRecord ,String tableName);
    int  delete(int dt,Long logtime,String tableName) ;
    void  truncate (String tableName);
    List<String> getSDKqid();
}
